app/lib/check_elastic_search.rb

```diff
@@ -37,13 +37,6 @@ def update_elasticsearch_doc(keys, data = {}, pm_id = nil, skip_get_data = false
     ElasticSearchWorker.perform_in(1.second, YAML::dump(model), YAML::dump(options), 'update_doc')
   end
 
-  def remove_fields_from_elasticsearch_doc(keys, pm_id)
-    return if self.disable_es_callbacks || RequestStore.store[:disable_es_callbacks]
-    options = { keys: keys, pm_id: pm_id }
-    model = { klass: self.class.name, id: self.id }
-    ElasticSearchWorker.perform_in(1.second, YAML::dump(model), YAML::dump(options), 'remove_fields')
-  end
```

Comment:
Instead of removing the field from the ES field, I update the value to nil. In the end, both approaches are the same, but I prefer updating the value to nil to be more consistent with other callbacks.